README.md

```diff
@@ -202,13 +202,17 @@ repeating the mistakes of the past.
 - Write blog posts or speak at meetups and conferences.
 
 #### Collaboration over competition
-- We have a very collaborative culture. There are a number of reasons why this is preferable:
-    - [Competitive cultures tend to not bode well for creating diverse teams.](https://medium.com/@racheltho/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#c63a).
+- We have a very collaborative culture. [Competitive cultures tend to not bode well for creating diverse teams](https://medium.com/@racheltho/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#c63a), which we value highly.
```

Comment:
> "We have a very collaborative culture ..."

Would it be more "motivating" if we changed this slightly to say something like 

> "Our engineering culture values collaboration over competition. We strive to build the most collaborative culture we can, because ...".

My point is, it might help to spell out that this is an ongoing effort to build on and improve that culture of collaboration.